core/include/detray/surface_finders/grid/detail/populator_impl.hpp

```diff
@@ -47,6 +49,20 @@ class bin_base {
         return view_type(m_content, args...);
     }
 
+    /// @returns view iterator over bin content in start position - non-const
```

Comment:
This makes the grid bins ranges and makes it possible to pick up them up with the join iterator